### ISSUE TITLE
Update to Android Studio 2021.1.1

### ISSUE THREAD

**Issue by samirotiv** [portapps/android-studio-portable#38]
Hi,

Thanks for your lovely work. I am looking forward to the build with Android Studio 2021.1.1. 

Could I please request you to make a build for it? Is there anything I can do to help? I will be happy to contribute.

Cheers!

**Comment by GithubUser96** [comment]
Yes I would also be grateful for an update. Some Gradle related features have been depreceated since and some built-in plugins are becoming increasingly incompatible with the latest builds of various apps making it significantly harder to build from source as you need to adapt the project to account for it. Please consider updating the release or merge the pull request made by samirotiv. Thanks.
